index.html

```diff
@@ -2085,6 +2095,39 @@ <h3>publicKeyHex</h3>
         </pre>
       </section>
 
+      <section id="publicKeyMultibase" about="https://w3id.org/security#publicKeyMultibase"
+      typeof="rdf:Property">
+        <h3>publicKeyMultibase</h3>
+        <p>
+The public key multibase property is used to specify the multibase-encoded
+version of a public key. Each key type definition is expected to specify
+one or more required encoding bases. Specifying only a single encoding
+base per key type is preferable as it reduces the burden to reach broad
+interoperability.
```

Comment:
I'm a bit worried that there's quite a bit of tribal knowledge in this. For example, how do I encode a nist p256 key here? How do I specify the encoding. If this is meant to be documentation to be "hey this thing exists" then this is fine, but if we're actually expecting people to build interoperable software against this definition they'll undoubtedly get stuck.

For specific action items I'd like to see, I'd like to see us reference how encoding works by directly referencing [multibase](https://github.com/multiformats/multibase#multibase-table) and additionally reference how [multicodec table](https://github.com/multiformats/multicodec/blob/master/table.csv#L86) is used and finally include how serialization of the bytes should work as well. 

I'm all for this additional change, but I think this is a bit too light to actually be something that someone without prior knowledge could implement.

Reply:
yes, probably also a good idea to always recommend compressed form keys.

Reply:
see also https://github.com/multiformats/multicodec/pull/210

Reply:
I have applied suggestions from @kdenhartog, @peacekeeper, and @OR13 in https://github.com/w3c-ccg/security-vocab/pull/103/commits/924eea44439a31ea2f22b3180664931559df8c62.

If you want further changes, please suggest concrete spec text -- it's hard to read minds and then write spec text. :P